api/turing/config/config.go

```diff
@@ -72,7 +73,8 @@ type Config struct {
 	//
 	// For example:
 	// { "experiment_engine_a": {"client": "foo"}, "experiment_engine_b": {"apikey": 12} }
-	Experiment map[string]interface{}
+	Experiment     map[string]interface{}
+	NewRelicConfig newrelic.Config
```

Comment:
Nitpick: Can we move this immediately above / below the Sentry config so that the similar configs are together in the type definition?

Reply:
Can we also add a simple test case for the config?

Reply:
yup done both.